File: Java/217_Contains_Duplicate/Solution.java
import java.util.HashSet;

public class Solution {
    public boolean containsDuplicate(int[] nums){
        HashSet<Integer> hashSet = new HashSet<Integer>();
        for(int index = 0; index < nums.length; index++){
            if(!hashSet.add(nums[index]))
                return true;
        }
        return false;
    }

    public static
}

File: JavaScript/217_Contains_Duplicate/solution.js
/**
 * @param {number[]} nums
 * @return {boolean}
 */
const containsDuplicate = function (nums) {
    const cache = new Set();
    for (const value of nums) {
        if (cache.has(value))
            return true;
        cache.add(value);
    }
    return false;
};
